libs/map/src/lib/components/action/vl-map-click-action-pindrop.ts

```diff
@@ -0,0 +1,68 @@
+import {webComponent} from '@domg-wc/common-utilities';
+import {css, LitElement} from "lit";
+
+
+/**
+ * VlMapClickActionPindrop
+ *
+ */
+@webComponent('vl-map-click-action-pindrop')
+export class VlMapClickActionPindrop extends LitElement {
+
+
+    // TODO: huisstijl vlaanderen?
+    // TODO: storybook element hiervoor nodig?
+    // Overgenomen van: https://hungyi.net/posts/pure-css-map-marker/
+    static styles = css`
+        :host {
+            /* Set the marker size here */
+            width: 2rem;
+            height: 2rem;
+            border-radius: 2rem;
+            /* Set the marker color here */
+            background: #aa3300;
```

Comment:
Hier heb je het rood overgenomen. Maar mss eerder blauw zoals Geopunt?
Aan UIG team om styling te bepalen zodat het cfr Vlaamse huisstijl is.

Reply:
heb er blauw van gemaakt. te bekijken @kspeltix